config/swagger.yaml

```diff
@@ -734,10 +738,11 @@ paths:
         - application/json
       parameters:
         - in: query
-          name: tx_objects
+          name: tx_encoding
           description: 'Transactions encoding: MessagePack (default) or JSON'
```

Comment:
etc...

Reply:
It should and it does. [3 lines bellow](https://github.com/aeternity/epoch/pull/644/files#diff-89b8ec7c2f28f3e3c7d21ca9bf070bb7R600):
```
enum: [message_pack, json]
```